src/plugins/unified_doc_viewer/public/plugin.tsx

```diff
@@ -26,11 +26,11 @@ const DocViewerTable = React.lazy(() => import('./components/doc_viewer_table'))
 const SourceViewer = React.lazy(() => import('./components/doc_viewer_source'));
 
 export interface UnifiedDocViewerSetup {
-  addDocView: DocViewsRegistry['addDocView'];
+  registry: DocViewsRegistry;
 }
 
 export interface UnifiedDocViewerStart {
-  getDocViews: DocViewsRegistry['getDocViewsSorted'];
+  registry: DocViewsRegistry;
```

Comment:
Liked the approach of exposing the registry 👍🏼 Makes it much cleaner and minimises future changes when new functions are added to the registry